intermine_boot/__init__.py

```diff
@@ -10,6 +10,7 @@
 
 
 @click.command()
+@click.version_option('0.0.2')
```

Comment:
This is a good simple test to start with, but like I mentioned during the call, it would be great if we didn't have to specify the version number in two places (here and in `setup.py`). Can you look into this before we merge?

Reply:
I have used pkg_resources module in setuptools.